### DISCUSSION TITLE
Disable codeLens?

### DISCUSSION THREAD

**Discussion by sofurihafe** [ocamllabs/vscode-ocaml-platform#825] [Q&A] [answered]
Is there a way to disable codeLens (types over definitions)?

**Comment by mlantas** [comment, marked as answer]
https://github.com/ocamllabs/vscode-ocaml-platform/issues/480#issuecomment-747106445

> There is a VSCode setting to disable it (from settings.json):
> 
> ```json
> "editor.codeLens": false
> ```
> 
> Or if you only want to disable it for OCaml:
> 
> ```json
> "[ocaml]": {
> 	"editor.codeLens": false
> }
> ```
> 
> You can also search for "code lens" in the VSCode settings tab and there will be a checkbox you can untick to disable it: ![image](https://user-images.githubusercontent.com/25037249/102419410-88664900-3fb4-11eb-9770-a2efc73c4a00.png)
